### ISSUE TITLE
Elevated right ventricular systolic pressure

### ISSUE THREAD

**Issue by tudorgroza** [obophenotype/human-phenotype-ontology#6084]
**Preferred term label:**
Elevated right ventricular systolic pressure

**Synonyms**
Elevated RVSP

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033119
